### PR TITLE
updated the night mode settings xml to have darkmode

### PR DESCRIPTION
**Pull request by PaulSonOfLars**

### CODE REVIEW

File: packages/SystemUI/res/xml/night_mode.xml [hunk 1]
```diff
@@ -1,12 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!-- Copyright (C) 2016 The Android Open Source Project
-
```
Comment (xdevs23) [inline packages/SystemUI/res/xml/night_mode.xml:3]: y u do dis
y u remove lines here

File: packages/SystemUI/res/xml/night_mode.xml [hunk 2]
```diff
@@ -1,12 +1,9 @@
...
      Licensed under the Apache License, Version 2.0 (the "License");
      you may not use this file except in compliance with the License.
      You may obtain a copy of the License at
-
```
Comment (xdevs23) [inline packages/SystemUI/res/xml/night_mode.xml:7]: .

File: packages/SystemUI/res/xml/night_mode.xml [hunk 3]
```diff
@@ -1,12 +1,9 @@
...
           http://www.apache.org/licenses/LICENSE-2.0
-
```
Comment (xdevs23) [inline packages/SystemUI/res/xml/night_mode.xml:9]: .

**Comment by PaulSonOfLars** [comment]
Wuuuut... Weird, and also why I hate git web

**Pull request merged**